xt-core-RC-0.1.0.md

```diff
@@ -0,0 +1,207 @@
+| Imagination DDK 1.11 KM | https://gitpct.epam.com/epmd-aepr/pvr_km_vgpu_img/commit/6994a4be87075975665fcef1fd95c087b0a007c8 |
+| Imagination DDK 1.15 KM | https://gitpct.epam.com/epmd-aepr/pvr_km_vgpu_img/commit/487a9326d5446bf4ed2808980ab0b12d8fd3f9e9 |
+| optee-os 3.9 | https://github.com/xen-troops/optee_os/commit/192b616c9715604a2d2b16bc61758d160550806e |
+| optee-test 3.9 | https://github.com/OP-TEE/optee_test/commit/21b347a3d75fd52fd49130e75c962c5b56123d2f |
+| linux 5.10.41 rcar-gen3 | https://github.com/xen-troops/linux/bf7d6bcdd618e5acb21118d87c76e5d8913f8dde |
+| linux 5.10.41 rcar-gen4 | https://github.com/xen-troops/linux/769ab722739878c3c1aaa1571f6ca996b135f8f6 |
+
+### meta-xt-rcar branch dunfell
+
+| component | repo and commit |
+| - | - |
+| Imagination DDK 1.11 UM | https://gitpct.epam.com/epmd-aepr/pvr_um_vgpu_img/commit/4ed61e604d925bfce8ab16645f48425a581496c7 |
+| Imagination DDK 1.11 KM | https://gitpct.epam.com/epmd-aepr/pvr_km_vgpu_img/commit/6994a4be87075975665fcef1fd95c087b0a007c8 |
+| optee-os 3.9 | https://github.com/xen-troops/optee_os/commit/192b616c9715604a2d2b16bc61758d160550806e |
+| optee-test 3.13 | https://github.com/OP-TEE/optee_test/commit/21b347a3d75fd52fd49130e75c962c5b56123d2f |
+| linux 5.10.41 rcar-gen3 | https://github.com/xen-troops/linux/d7c1b717e3f11b6aa0713dcefa8cf1ed4c1222af |
+| linux 5.10.41 rcar-gen4 | https://github.com/xen-troops/linux/769ab722739878c3c1aaa1571f6ca996b135f8f6 |
+
+
+## Variables required to be set by the product
+
+Note that `%{SOMETHING}` represents some value that is defined by the product and is transferred into `xt-core` through yocto variables.
+
+See the following yaml-files as working reference examples.
+For PV-based configuration - https://github.com/xen-troops/meta-xt-prod-devel-rcar/blob/master/prod-devel-rcar.yaml
+For virtio-based configuration - https://github.com/xen-troops/meta-xt-prod-devel-rcar/blob/master/prod-devel-rcar-virtio.yaml
+
+### Variables required to be set for Dom0
+```
+MACHINE = "%{DOM0_MACHINE}"
+XT_DOMD_CONFIG_NAME = "%{XT_DOMD_CONFIG_NAME}"
+XT_DOMD_DTB_NAME = "%{XT_DOMD_DTB_NAME}"
+XT_DOM_NAME = "dom0"
+XT_GUEST_INSTALL = "domd %{XT_GENERIC_DOMU_TAG} %{XT_DOMA_TAG} %{XT_DOMZ_TAG}"
+INIT_MANAGER = "systemd"
+```
+
+If DomU is present
+```
+XT_DOMU_DTB_NAME = "%{XT_DOMU_DTB_NAME}"
+XT_DOMU_CONFIG_NAME = "%{XT_DOMU_CONFIG_NAME}"
+```
+
+If DomA is present
+```
+XT_DOMA_DTB_NAME = "%{XT_DOMA_DTB_NAME}"
+XT_DOMA_CONFIG_NAME = "%{XT_DOMA_CONFIG_NAME}"
+OVERRIDES:append = ":enable_android"
+```
+
+If DomZ is present
+```
+XT_DOMZ_CONFIG_NAME = "%{XT_DOMZ_CONFIG_NAME}"
+```
+
+### Variables required to be set for DomD
+```
+MACHINE = "%{DOMD_MACHINE}"
+XT_DOM_NAME = "domd"
+XT_OP_TEE_FLAVOUR = "%{XT_OP_TEE_FLAVOUR}"
+XT_DEVICE_TREES = "%{XT_DOMD_DTB_NAME} %{XT_DOMA_DTB_NAME} %{XT_XEN_DTB_NAME}"
+XT_GUEST_INSTALL = "%{XT_GENERIC_DOMU_TAG} %{XT_DOMA_TAG} %{XT_DOMZ_TAG}"
+XT_MULTIMEDIA_EVA_DIR = "%{XT_MULTIMEDIA_EVA_DIR}"
```

Comment:
That's very Gen3 specific, I guess it needs some comment about this

Reply:
Thanks. I have added info for multimedia and graphic packages, that they are R-Car Gen3 specific proprietary packages, and are not a part of the release.